sdcm/tester.py

```diff
@@ -1178,8 +1178,8 @@ def verify_stress_thread(self, cs_thread_pool):
             self.log.warning('There is no stress results, probably stress thread has failed.')
         errors = errors[-5:]
         if errors:
-            self.fail("cassandra-stress errors on "
-                      "nodes:\n%s" % "\n".join(errors))
+            self.log.warning("cassandra-stress errors on "
```

Comment:
if we change this the test will never stop due to errors in the C-S, since all events have severity Error:
```
CassandraStressLogEvent(type='IOException', regex=r'java\.io\.IOException', severity=Severity.ERROR),
CassandraStressLogEvent(type='ConsistencyError', regex=r'Cannot achieve consistency level', severity=Severity.ERROR)
```
@roydahan do we want this?

Reply:
did this fail catch an event (`CassandraStressLogEvent`) or an error in the loader log?

Reply:
so, how could we then not fail a test, when it is acceptable to have `IOException`?

Reply:
@bentsi I thought we have the mechanize to stop the test by looking at the events, regardless to this one.
so I think we need to decide which part would stop the test, `verify_stress_thread` or the event that would raise out of it.

Reply:
Of course not approved.

Reply:
@roydahan I've lost you, what do you mean, "not approved" in this context ?

Reply:
If it means that we won't stop the test on IOException like @bentsi commented above, this change is not approved.

Reply:
Again we have two places a test would stop, 
* in case cassandra stress ended with a failure (no this case), and test would stop as the stress failed.
* in this case test would fail only a the end of the stress command, in a regular longevity that means practically at the end of the test, and not immediately as it happened, test would fail in the end regardless, since we had an event with level ERROR. so the problem would be noticed and the whole job would mark as failure

in this case, we filter the ERROR event, but the code reading the log at the end of cassandra-stress (which is there from ages ago before we have the event system) is failing the test, and we can't control it, even if this failure is expected (i.e. in this case)

I think it would be safe enough to remove this extra check of the log at the end of c-s run, since we are reading the log while it running an emit events, code in `verify_results` checking the output for again, can now be safely removed:
```python
                if 'java.io.IOException' in line:
                    errors += ['%s: %s' % (node, line.strip())]
```